include/crm/common/actions.h

```diff
@@ -16,13 +16,7 @@
 #include <libxml/tree.h>                // xmlNode
 
 #include <crm/lrmd_events.h>            // lrmd_event_data_t
-
-#include <glib.h>                       // GList, GHashTable
-#include <libxml/tree.h>                // xmlNode
```

Comment:
still have `xmlNode`s

Reply:
it was included twice :)

Reply:
ah, sure was. The diff wasn't showing the lines above these by default.